x-pack/plugins/observability_solution/apm/public/components/app/service_groups/service_group_save/select_services.tsx

```diff
@@ -117,7 +119,7 @@ export function SelectServices({
     <Container>
       <EuiModalHeader>
         <div>
-          <EuiModalHeaderTitle>
+          <EuiModalHeaderTitle id={titleId}>
```

Comment:
Do you need to receive it as property? Can't you just call `useGeneratedHtmlId` here?

```suggestion
          <EuiModalHeaderTitle id={useGeneratedHtmlId()}>
```

Reply:
Not sure, please see [service_group_save/save_modal.tsx](https://github.com/elastic/kibana/pull/194696/files#diff-9c52808164477342bf3b26933eda0c3f9091007ecb66e4c4775157c7b5f1999b) file . It's a parent Component where we also should set the aproppriate value in  `aria-labelledby`